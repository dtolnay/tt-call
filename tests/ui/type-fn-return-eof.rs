use tt_call::{parse_type, tt_call};

tt_call! {
    macro = [{ parse_type }]
    input = [{ fn() -> }]
}

fn main() {}
