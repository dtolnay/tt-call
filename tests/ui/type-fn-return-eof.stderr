error: no rules expected the token `->`
 --> $DIR/type-fn-return-eof.rs:5:21
  |
5 |     input = [{ fn() -> }]
  |                     ^^ no rules expected this token in macro call
